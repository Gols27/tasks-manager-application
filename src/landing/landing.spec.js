import React from "react";
import Landing from "./";
import { mount } from "enzyme";
import { mountToJson } from "enzyme-to-json";

it("renders without crashing", () => {
  const wrapper = mount(<Landing />);
  expect(mountToJson(wrapper)).toMatchSnapshot();
});
