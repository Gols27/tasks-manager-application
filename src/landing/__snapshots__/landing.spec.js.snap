// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<Landing>
  <AppLoader>
    <div
      className="loader-container"
    >
      <div
        className="loader"
      />
    </div>
  </AppLoader>
</Landing>
`;
